### ISSUE TITLE
Trouble with initial VoxelMorph setup

### ISSUE THREAD

**Issue by dnschouten** [voxelmorph/voxelmorph#210]
Hello!

First of all, thank you for your great work! I am currently following the learn2reg tutorial step-by-step as practice. I am however having some trouble with the initial setup.

**Problem 1.**
When I try to _import neuron_, I obtain a 

> "ModuleNotFoundError: No module named 'pytools.patchlib'". 

As mentioned on the VoxelMorph front page, I made sure to have each folder in the ext folder on my python path. However, for some reason pytools is not recognized as a module while pynd is. 

**Problem 2.**  
I'm able to _import voxelmorph as vxm_ without errors, but when I try to create a unet by calling 
_unet = vxm.networks.unet_core(vol, enc_num, dec_num)_, I get the following error:

> "AttributeError: module 'voxelmorph' has no attribute 'networks'"

Could you give me some pointers on what I might be doing wrong? Thanks in advance!

**Comment by adalca** [comment]
@dnschouten  sorry we missed this -- did you manage to get it working, or are you still having trouble. 
It seems like `PYTHON_PATH` issues to me. 

Also, this was right around the time we were pushing new code to the `master` branches of both `neuron` and `voxelmorph`, so it was a bit hard to follow issues then :)

**Comment by dnschouten** [comment]
No worries, I managed to get it working in the meantime. As for the first problem, I found out that I had previously installed a different pytools package from a different project. Simply uninstalling that one solved the problem. 

For the second problem I couldn't find the culprit, so I ended up just adding the \src directory to the python path and `import networks` directly. Perhaps not the most pythonic way but it works fine for now. Thanks for the reply!
